New Group/HomeScreen/ListMovieViewController.swift

```diff
@@ -0,0 +1,123 @@
+import UIKit
+
+fileprivate struct DefinesHeight {
+    static let movieTableRowHeight: CGFloat = 138
+}
```

Comment:
this struct should be set inside at top of the class ListMovieViewController and renamed "Defines" or "Constants"